examples/dhf-flow/build.gradle

```diff
@@ -304,9 +279,8 @@ task runPersonMastering(type: com.marklogic.gradle.task.client.CallResourceTask)
   body = []
   params = [
     'collector-name': 'collect',
-    'collector-name': 'collect',
-    'collector-ns': 'http://marklogic.com/data-hub/plugins',
-    'collector-at': '/entities/MDM/harmonize/SmartMaster/collector.xqy',
+    'collector-ns': 'http://marklogic.com/smart-mastering/collector',
```

Comment:
Added a URI collector that will work for most uses to the core code library and referencing it here.